### ISSUE TITLE
Latest version of `rich` breaks pyparam usage

### ISSUE THREAD

**Issue by jonEbird** [pwwang/pyparam#14]
Looks like the latest version of rich has a [breaking change](https://github.com/Textualize/rich/blob/5f0219786bbaf994061011df087602d5edff5d66/CHANGELOG.md#1100---2022-01-09) that client will need to adjust to. Locally you can expect to see:
```raw
...
  File "/Users/jonmiller/mycli/.venv/lib/python3.8/site-packages/pyparam/help.py", line 12, in <module>
ImportError: cannot import name 'RenderGroup' from 'rich.console'
```

**Comment by pwwang** [comment]
Do you mean rich v11.0.0?

**Comment by jonEbird** [comment]
Yes. Locally I've restricted `rich` to `rich>=9.5.1,<11.0.0` within my requirements.txt file. If `rich==11.0.0`, then pyparam exception occurs.

**Comment by pwwang** [comment]
Fixed at version [0.4.17](https://github.com/pwwang/pyparam/releases/tag/0.4.17)
